rskj-core/build.gradle

```diff
@@ -204,12 +203,13 @@ task javadocJar(type: Jar) {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
why do we need this?

Reply:
Because build fails saying it finds duplicated `build-info.properties` and it doesn't know what to do with it when building the jar. I think this happens due to the version upgrade

Reply:
ok. I see. The thing is that we do need some duplicates, for instance, license files. See [here](https://github.com/rsksmart/rskj/blob/master/rskj-core/build.gradle#L291). One thing you can do is as it's done [here](https://github.com/rsksmart/rskj/blob/master/rskj-core/build.gradle#L251).